src/routes/Home.tsx

```diff
@@ -37,20 +40,21 @@ import { homeCommandBarConfig } from 'lib/commandBarConfigs/homeCommandConfig'
 const Home = () => {
   const { commandBarSend } = useCommandsContext()
   const navigate = useNavigate()
-  const { projects: loadedProjects, newDefaultDirectory } =
+  const { projects: loadedProjects /*newDefaultDirectory*/ } =
     useLoaderData() as HomeLoaderData
   const {
     settings: {
       context: { defaultDirectory, defaultProjectName },
       send: sendToSettings,
     },
   } = useGlobalStateContext()
-  if (newDefaultDirectory) {
-    sendToSettings({
-      type: 'Set Default Directory',
-      data: { defaultDirectory: newDefaultDirectory },
-    })
-  }
+  // TODO figure out why newDefaultDirectory isn't on HomeLoaderData anymore
+  // if (newDefaultDirectory) {
+  //   sendToSettings({
+  //     type: 'Set Default Directory',
+  //     data: { defaultDirectory: newDefaultDirectory },
+  //   })
+  // }
```

Comment:
@franknoirot when I merged in main I started getting types errors because newDefaultDirectory is not on HomeLoaderData anymore. Do you know why that might be?